tracer/test/Datadog.AutoInstrumentation.NativeLoader.Tests/runtimeid_store.cpp

```diff
@@ -0,0 +1,39 @@
+#include "gtest/gtest.h"
+
+#include "../../src/Datadog.AutoInstrumentation.NativeLoader/runtimeid_store.h"
+
+using namespace datadog::shared::nativeloader;
+
+TEST(runtimeid_store, SameIdIfAddMoreThanOnce)
+{
+    AppDomainID id = 42;
+    RuntimeIdStore store;
+    store.Generate(id);
+
+    auto rid = store.Get(id);
+
+    store.Generate(id);
+    auto anotherRid = store.Get(id);
+
+    ASSERT_EQ(rid, anotherRid);
```

Comment:
Invert the result based on my previous comment (i.e. the GUID should always be different)

Reply:
✔️